addons/web/static/src/views/fields/properties/property_definition.scss

```diff
@@ -45,7 +33,7 @@
         }
     }
 
-    td {
+    td:not(:last-of-type) {
         padding-bottom: 15px;
```

Comment:
since we're here, better to replace `15px` with something like `map-get($spacers, 3)` or smthg...